pkgs/development/python-modules/googletrans/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi, requests, pytest, coveralls }:
+
+buildPythonPackage rec {
+  pname = "googletrans";
+  version = "2.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1iack18jc1ahpl8axv06aah5by05ff5j653ad478f91awxp2lkmv";
+  };
+
+  propagatedBuildInputs = [
+    requests
+  ];
+
+  doCheck = true;
+  checkInputs = [ pytest coveralls ];
```

Comment:
looks like no tests, please checkout from https://github.com/ssut/py-googletrans

```
running build_ext

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
Finished executing setuptoolsCheckPhase
pytestcachePhase
derivation '/nix/store/kd1waqmbiiks11rpfabg85w2nnn4wzw4-python2.7-googletrans-2.4.0.drv'
```

Reply:
Well spotted! Will do.

Reply:
Had to add an explicit `checkPhase`. Tests now running and passing.